csvqb/csvqb/writers/qbwriter.py

```diff
@@ -54,13 +57,16 @@ def write(self, output_folder: Path):
         # todo: Local units haven't been defined anywhere yet!
         # todo: Unit multiplier functionality hasn't been output.
 
+        see_also = rdf_resource_to_json_ld(self._generate_qb_dataset_dsd_definitions())
+        for x in self._serialise_attribute_values():
+            see_also += rdf_resource_to_json_ld(x)
+
```

Comment:
You need to make sure you serialise the units here too!

Reply:
unit serialization is added